concordium-std/src/traits.rs

```diff
@@ -348,6 +349,16 @@ pub trait HasHost<State>: Sized {
         self.invoke_contract_raw_read_only(to, Parameter(&param), method, amount)
     }
 
+    /// Get the current exchange rates for Euro per Energy and micro CCD per
+    /// Euro.
+    fn exchange_rates(&self) -> ExchangeRates;
+
+    /// Get the current balance of an account.
+    fn account_balance(&self, address: AccountAddress) -> QueryAccountBalanceResult;
+
+    /// Get the current balance of a contract instance.
```

Comment:
And here. The case of the balance of the contract's own address is rather ambiguous.

Reply:
I have added more to this

Reply:
It is better now.

But the best would be if we had a simple 
```
let balance_before = host.account_balance(account)?;
host.transfer(...,amount,...)?;
let balance_after = host.account_balance(account)?;
// balance_after = balance_before + amount
```

Reply:
I have added this to both contract_balance and account_balance